src/components/data/LocalVariablesContent.js

```diff
@@ -236,7 +298,8 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
           Utils.withBusyState(setBusy)
         )(async () => {
           setDeleteIndex()
-          await Ajax().Workspaces.workspace(namespace, name).deleteAttributes([amendedAttributes[deleteIndex][0]])
+          await Ajax().Workspaces.workspace(namespace, name).deleteAttributes([amendedAttributes[deleteIndex][0],
+            toDescriptionKey(amendedAttributes[deleteIndex][0])])
```

Comment:
This might be easier to read if we tweak where the linebreak lands, what do you think?
```suggestion
          await Ajax().Workspaces.workspace(namespace, name)
            .deleteAttributes([amendedAttributes[deleteIndex][0], toDescriptionKey(amendedAttributes[deleteIndex][0])])
```

Reply:
Done.